src/sentry/lang/javascript/processor.py

```diff
@@ -388,7 +388,7 @@ def fetch_file(url, project=None, release=None, dist=None, allow_scraping=True):
     # For JavaScript files, check if content is something other than JavaScript/JSON (i.e. HTML)
     # NOTE: possible to have JS files that don't actually end w/ ".js", but
     # this should catch 99% of cases
-    if url.endswith(".js"):
+    if urlsplit(url).path.endswith(".js"):
```

Comment:
Would catch down ValueError if this is unparseable

Reply:
I don't think it's possible for this code to throw, as we use the very same URL for fetching the content. So by the time we are here, it had to be valid for making an actual request.